plugins/pyembed/pyembed.cpp

```diff
@@ -274,6 +274,8 @@ static class Python27GlobalState
 #endif
         // Initialize the Python Interpreter
         Py_Initialize();
+        const char *argv[] = { "ecl", nullptr} ;
```

Comment:
Should the 1st parameter be a null string - the documentation implies it will search for a script file with that name.

Also, should it be a static variable?
Should it be calling PySys_SetArgvEx?